src/src/NotifierStructs/N001_data_struct.cpp

```diff
@@ -341,13 +350,14 @@ bool NPlugin_001_send(const NotificationSettingsStruct& notificationsettings, St
           myStatus = true;
         }
 
-        NPlugin_001_MTA(client, F("QUIT"), 221, clientTimeout); // Sent successfully, close SMTP protocol, ignore failure
+        NPlugin_001_MTA(*client, F("QUIT"), 221, clientTimeout); // Sent successfully, close SMTP protocol, ignore failure
         break;
       }
     }
 
     //    client.PR_9453_FLUSH_TO_CLEAR();
-    client.stop();
+    client->stop();
+    delete client;

```

Comment:
This is in the scope of `if (!failFlag)`, so in theory this delete may not be performed.
Should be done right before the `return`

Reply:
Moved.